### DISCUSSION TITLE
Readme.md - add mirror for older versions of Audacity?

### DISCUSSION THREAD

**Discussion by Darthagnon** [tenacityteam/tenacity-legacy#269] [General]
I saw [this post on Reddit](https://www.reddit.com/r/DataHoarder/comments/oe2opu/due_to_the_new_audacity_terms_of_service_i/), where a user archived a lot of older versions of Audacity on [Archive.org](https://archive.org/details/AudacityArchive), because the Audacity team, allegedly, "have in the past pulled prior versions". 

As the readme has a section dedicated to older versions, linking to the original repo, would it be worth adding a link/mention of this archive as a primary mirror for older Audacity builds?

I don't know if there's a way of verifying their authenticity.

**Comment by poyribn** [comment]
Yes, this archive.org link is a worthwhile one to reference.

> I don't know if there's a way of verifying their authenticity.

No problem. Just state something like this:
> _Although we could not verify its authenticity_, we also found [this archive of the pre-fork versions of Audacity](>Insert the Link<)

**Reply by Pythone** [reply]
I agree. Also some forks have built in archives
